### DISCUSSION TITLE
Enhancements in utilities app

### DISCUSSION THREAD

**Discussion by fatmanawfal** [EnabApp/nuxt3#101] [Ideas]
**Hi @william-iq397 and @i1Zeus** 
Be interested in developing applications, I suggest some modifications such as
1. add history icon in small size
2. remove the result of the calculators after I close the calculator, if you notice that the result is not deleted after exiting the application
3. The equal button should not work if there is single number
4. the calculator app flexibility but put in your mind the to-do app should be more flexibility than calculator

give me your opinions.

**Comment by i1Zeus** [comment]
### sup!
Ok, tbh I already have these in mind, didn't have enough time because I was on a time schedule and as u know the last calc app **not as good as we wanted** ... but about the result... I like to call it **QAH** aka **Quick Access History**, the reason, what if u closed the app and u forgot the result?... **BOOM!** it's there or what if u accidently  closed the app? seeing the result in the calc screen might and will save u some time!

**Reply by fatmanawfal** [reply]
you are right, I liked **QAH**
